ts/parts/SVGElement.ts

```diff
@@ -844,7 +846,10 @@ class SVGElement {
             if (complete) {
                 animOptions.complete = complete;
             }
-            animate(this as any, params, animOptions);
+            // If defer option is defined delay the animation #12901
+            syncTimeout((): void => {
```

Comment:
Inside the timeout, we should test if the element still exists: https://jsfiddle.net/highcharts/zLrkosq0/1/